src/DssSpell.t.base.sol

```diff
@@ -2370,4 +2540,228 @@ contract DssSpellTestBase is Config, DssTest {
+        _scheduleWaitAndCast(address(spell));
+        assertTrue(spell.done());
+
+        assertEq(vow.flapper(), address(split), "TestError/invalid-vow-flapper");
+        assertEq(split.flapper(), address(flap), "TestError/invalid-split-flapper");
+        assertEq(flap.gem(), address(sky), "TestError/invalid-flapper-gem");
+        assertEq(flap.pip(), addr.addr("FLAP_SKY_ORACLE"), "TestError/invalid-flapper-pip");
+        assertEq(flap.pair(), addr.addr("UNIV2USDSSKY"), "TestError/invalid-flapper-pair");
+
+        // Check splitter and flapper
+        {
+            // Leave surplus buffer ready to be flapped
+            vow.heal(vat.sin(address(vow)) - (vow.Sin() + vow.Ash()));
+            // Ensure flapping is possible
+            stdstore
+                .target(address(vat))
+                .sig("dai(address)")
+                .with_key(address(vow))
+                .checked_write(vat.sin(address(vow)) + vow.bump() + vow.hump());
+
+            GemAbstract pair = GemAbstract(addr.addr("UNIV2USDSSKY"));
+            FlapOracleLike pip = FlapOracleLike(flap.pip());
+
+            vm.prank(address(flap));
+            uint256 price = uint256(pip.read());
+
+            // Ensure there is enough liquidity
+            uint256 usdsWad = 150_000_000 * WAD;
+            GodMode.setBalance(address(usds), address(pair), usdsWad);
+            // Ensure price is within the tolerane (flap.want() + delta (1 p.p.))
+            uint256 skyWad = usdsWad * (flap.want() + 10**16) / price;
+            GodMode.setBalance(address(sky), address(pair), skyWad);
+
+            uint256 lotRad = vow.bump() * split.burn() / WAD;
+            uint256 payWad = (vow.bump() - lotRad) / RAY;
+
+            uint256 pbalancePauseProxy = pair.balanceOf(pauseProxy);
+            uint256 pdaiVow = vat.dai(address(vow));
+            uint256 preserveUsds = usds.balanceOf(address(pair));
+            uint256 preserveSky = sky.balanceOf(address(pair));
+
+            uint256 pbalanceUsdsFarm;
+            // Checking the farm balance is only relevant if split.burn() < 100%
+            if (split.burn() < 1 * WAD) {
+                pbalanceUsdsFarm = usds.balanceOf(split.farm());
+                assertFalse(split.farm() == address(0), "TestError/Splitter/missing-farm");
+            }
+
+            vow.flap();
+
+            assertGt(pair.balanceOf(pauseProxy),      pbalancePauseProxy,   "TestError/Flapper/pair-pauseProxy-balance-no-increase");
+            assertGt(usds.balanceOf(address(pair)),   preserveUsds,         "TestError/Flapper/usds-pair-balance-no-increase");
+            assertEq(sky.balanceOf(address(pair)),    preserveSky,          "TestError/Flapper/unexpected-sky-pair-balance-change");
+            assertGt(pdaiVow - vat.dai(address(vow)), vow.bump() * 9 / 10,  "TestError/Flapper/vat-dai-vow-change-too-low");
+            assertLt(pdaiVow - vat.dai(address(vow)), vow.bump() * 11 / 10, "TestError/Flapper/vat-dai-vow-change-too-high");
```

Comment:
This should be just `assertEq` now as this version of the `flapper` (built for the `splitter`) uses the `lot` as the total and calculates the amount to `sell` from there without adding anything on top.
Btw this situation brings an extra thing, as now the `vow.bump` is the total and not just the `sell` part, in order to keep the same behavior, the value should be duplicated or `hop` reduced to half.

Reply:
Already discussed, only the assertEq thing is relevant.